### PR TITLE
feat(shortcut): add keyboard shortcut for gpt-block

### LINKED ISSUE #34: Keyboard shortcuts

### PR DESCRIPTION
**Pull request by xxchan**
Only added shortcut for `/gpt-block` because I think it is the most frequently used one 🤔

fix #34

### CODE REVIEW

**Comment by briansunter** [comment]
Hey @xxchan this looks really good but would you mind defaulting it to something besides mod+g? I'm using that shortcut on an upcoming feature

File: src/main.ts [hunk 1]
```diff
@@ -51,6 +51,13 @@ const settingsSchema: SettingSchemaDesc[] = [
     description:
       "Size of the image to generate. Can be 256, 512, or 1024. Smaller images are faster to generate.",
   },
+  {
+    key: "shortcutBlock",
+    type: "string",
+    default: "mod+g",
```
Comment (xxchan) [inline src/main.ts:57]:
```suggestion
    default: "",
```
@briansunter Could you suggest another one? Or maybe simply remove the default. :)

Comment (briansunter) [reply src/main.ts:57]: Hey @xxchan I still like the idea of having a default. I don't mind what the shortcut is, I don't know off the top of my head which ones are already being used by logseq. I think as long as it's not taken by the built in logseq shortcuts it should be fine.

**Review by briansunter — APPROVED** [review]
Just tested it and it looks good. Thank you for the contribution!

**Pull request merged**